test/Integration/tpp-run-splat-tensor.mlir

```diff
@@ -39,9 +51,44 @@ func.func @entry(%input: tensor<4x2xf32>) {
 // RANDOM-SPLAT-LABEL: @_entry
 // RANDOM-SPLAT-NOT: arith.constant dense<1.000000e+00> : tensor<2x16xf32>
 // RANDOM-SPLAT-NOT: arith.constant dense<2.000000e+00> : tensor<4x16xf32>
-// RANDOM-SPLAT-NOT: arith.constant dense<0.000000e+00> : tensor<4x8xf32>
+// RANDOM-SPLAT: arith.constant dense<0.000000e+00> : tensor<4x8xf32>
 // RANDOM-SPLAT: arith.constant dense<{{.*}}0.000000e+00, 1.000000e+00], [2.000000e+00, 3.000000e+00{{.*}}>
 // RANDOM-SPLAT: arith.constant dense<0> : tensor<4x8xi32>
 // Input
 // RANDOM-SPLAT-LABEL: @entry
 // RANDOM-SPLAT-NOT: arith.constant dense<1.000000e+00> : tensor<4x2xf32>
+
+// OPT-CONST-LABEL: @_entry
+// OPT-CONST: arith.constant dense<1.000000e+00> : tensor<2x16xf32>
+// OPT-CONST: arith.constant dense<1.000000e+00> : tensor<4x16xf32>
+// OPT-CONST: arith.constant dense<0.000000e+00> : tensor<4x8xf32>
+// OPT-CONST: arith.constant dense<{{.*}}0.000000e+00, 1.000000e+00], [2.000000e+00, 3.000000e+00{{.*}}>
+// OPT-CONST: arith.constant dense<0> : tensor<4x8xi32>
+
+// OPT-SIMPLE-LABEL: @_entry
+// OPT-SIMPLE: arith.constant dense<{{.*}}3.000000e-01, 6.000000e-01, 0.899999976, {{.*}}> : tensor<2x16xf32>
+// OPT-SIMPLE: arith.constant dense<{{.*}}3.000000e-01, 6.000000e-01, 0.899999976, {{.*}}> : tensor<4x16xf32>
+// OPT-SIMPLE: arith.constant dense<0.000000e+00> : tensor<4x8xf32>
+// OPT-SIMPLE: arith.constant dense<{{.*}}0.000000e+00, 1.000000e+00], [2.000000e+00, 3.000000e+00{{.*}}>
+// OPT-SIMPLE: arith.constant dense<0> : tensor<4x8xi32>
+
+// OPT-CONT-LABEL: @_entry
+// OPT-CONT: arith.constant dense<{{.*}}0xFFC00000, 0x7F800000, 0x7F800000, {{.*}}> : tensor<2x16xf32>
```

Comment:
Does this work correctly? It doesn't look continuous.

Reply:
Nope, it was initialized with `i/0`

Reply:
Yup, I did not notice! :open_mouth: